src/events/command-handler.ts

```diff
@@ -144,6 +141,12 @@ export class CommandHandler implements EventHandler {
         } catch (error) {
             await this.sendError(intr, data);
 
+            let args = intr.options.data
+                .map(option => {
+                    return `${option.name}: ${option.value}`;
```

Comment:
Just trying to think, is option.value every something really long? Like what about an attachment?